File: ecommerce-backend/Models/Supplier.cs
using System;
using System.Collections.Generic;

namespace ecommerce_backend.Models;

public partial class Supplier
{
    public int SupplierId { get; set; }

    public string? Name { get; set; }

    public string? Email { get; set; }

    public string? Phone { get; set; }

    public string? Address { get; set; }

    public bool? Status { get; set; }

    public string? Notes { get; set; }

    public virtual ICollection<Product> Products { get; set; } = new List<Product>();
}

File: ecommerce_frontend/src/Models/Supplier.ts
export type SupplierGet = {
    supplierId: number;
    name: string;
    notes: string;
    phone: string;
    status: boolean;
    email: string;
    address: string;
}

export type SupplierPost = {
    name: string;
    email: string;
    phone: string;
    address: string;
    status: boolean;
    notes: string;
}

export type SupplierPut = {
    name: string;
    email: string;
    phone: string;
    address: string;
    status: boolean;
    notes: string;
}